Added PHP sample to connect to UNICAS

amisls opened the pull request.

akoval-liaison commented on php/php-sample-cas-api-client-master/test.php:
```diff
@@ -0,0 +1,11 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';  // <---- THIS IS REQUIRED
+
+$dotenv = Dotenv\Dotenv::createMutable(__DIR__);
+$dotenv->load();
+
+echo 'CAS_API_BASE_URI = ' . getenv('CAS_API_BASE_URI') . PHP_EOL;
+echo 'From getenv: ' . getenv('CAS_API_BASE_URI') . PHP_EOL;
+echo 'From _ENV: ' . ($_ENV['CAS_API_BASE_URI'] ?? 'not set') . PHP_EOL;
+echo 'From _SERVER: ' . ($_SERVER['CAS_API_BASE_URI'] ?? 'not set') . PHP_EOL;
```
Should this file be present in examples?

amisls committed:
Added php sample to connect to UNICAS.

Added php sample to connect to UNICAS.

Added php sample to connect to UNICAS.

Added php sample to connect to UNICAS.
